core/src/proc_macros_support.rs

```diff
@@ -0,0 +1,23 @@
+use jsonrpsee_types::ErrorObjectOwned;
+
```

Comment:
Can you add a comment why cold is used here? 

I confuse it with "inlining" would be good explain why it used here...

Reply:
We're marking functions on the error path as `#[cold]` to both reduce chance of inlining (which is what I'm trying to prevent with this PR) and to make the generated assembly slightly better.

I'll add a comment

Reply:
thanks